com.unity.render-pipelines.core/Editor/Debugging/DebugWindow.cs

```diff
@@ -423,18 +433,16 @@ void OnGUI()
                 Rect splitterRect = new Rect(splitterPos - 3, 0, 6, Screen.height);
                 GUI.Box(splitterRect, "", s_SplitterLeft);
 
-                GUILayout.Space(10f);
+                GUILayout.Space(2f);
```

Comment:
Maybe these indentation constants should be a group of named constants, a sort of "theme" maybe.

Reply:
I added a name for these so they are easier to understand and change. For a broader theme, I think it might just end up looking like a garbled mess of arcane names given to every integer between 0 and 20 :) after all, to be consistent it should be done for the numbers 3 and 6 visible above as well, but many of these numbers just come down to trying to match the UI design so it looks right...